ouroboros-consensus-cardano/tools/db-analyser/Main.hs

```diff
@@ -326,35 +116,80 @@ parseAnalysis = asum [
           long "show-ebbs"
         , help "Show all EBBs and their predecessors"
         ]
+    , pure OnlyValidation
     ]
 
+blockTypeParser :: Parser BlockType
+blockTypeParser = subparser $ mconcat
+  [ command "byron"   (info (parseByronType   <**> helper) (progDesc "byron command"  ))
+  , command "shelley" (info (parseShelleyType <**> helper) (progDesc "shelley command"))
+  , command "cardano" (info (parseCardanoType <**> helper) (progDesc "cardano command"))
```

Comment:
I'm trying out the tool and with `--help`, it looks like this:
```
[..]
Available commands:
  byron                    byron command
  shelley                  shelley command
  cardano                  cardano command
```
Better would be:
```
[..]
Available commands:
  byron                    Analyse a Byron-only DB
  shelley                  Analyse a Shelley-only DB
  cardano                  Analyse a Cardano DB
```